ui/Screen/NetworkDiagnostics/StateTable.svelte

```diff
@@ -33,10 +36,9 @@
               {#each Object.keys(state[urn].peers).sort() as peerId}
                 <tr>
                   <td>
-                    <StyledCopyable
-                      value={peerId}
-                      truncate={true}
-                      expandable={false} />
+                    <Copyable title="Device ID" clipboardContent={peerId}>
```

Comment:
How about using the `PeerId` component here? Maybe with an `icon=false` attribute. This would eliminate some duplication and the need to extract the presentation in a separate file.

Reply:
c4b23c1c51d798db78baeb055b138589ce938436